### ISSUE TITLE
Up/Down Arrow key navigation

### ISSUE THREAD

**Issue by craigkg** [kenwheeler/slick#2070]
I'm working on a navigational system consisting of a horizontal slider that within each slide contains a vertical slider. For accessibility reasons, I must use the keyboard arrow keys for navigation with the mouse essentially disabled. With accessibility set to true, the up and down arrow keys do not navigate the vertical slider, however the left and right arrow keys trigger the horizontal slider navigation and the navigation of the first vertical slider (or whichever is in focus). Can the vertical sliders be set to accept up arrow key/down arrow key input for navigation rather than the left/right arrow keys?

example: http://jsfiddle.net/craigkg/sxymocr6/1/

**Comment by M4urice** [comment]
Dirtiest workaround: Change slack.js `event.keyCode` according to your needs (https://github.com/kenwheeler/slick/blob/master/slick/slick.js#L1437):
````
 if (!event.target.tagName.match('TEXTAREA|INPUT|SELECT')) {
            if (event.keyCode === 38 && _.options.accessibility === true) {
                _.changeSlide({
                    data: {
                        message: _.options.rtl === true ? 'next' : 'previous'
                    }
                });
            } else if (event.keyCode === 40 && _.options.accessibility === true) {
                _.changeSlide({
                    data: {
                        message: _.options.rtl === true ? 'previous' : 'next'
                    }
                });
            }
        }
````

Key code 38 and 40 are up and down (Check http://keycode.info/ for others).


Also you may want disable the default scrolling in the browser on these events:
````
window.addEventListener("keydown", function(e) {
    // space and arrow keys
    if([32, 37, 38, 39, 40].indexOf(e.keyCode) > -1) {
        e.preventDefault();
    }
}, false);
````
http://stackoverflow.com/a/8916697/2175370
